docs/digger/developer/jenkinsfile.asciidoc

```diff
@@ -0,0 +1,166 @@
+== Using Jenkinsfiles
+
+=== Checkout source code
+To checkout a projects source code the `checkout` step can be used.
+
+[source,groovy]
+----
+node("android"){
+  ...
+  stage("Checkout"){
+    checkout scm
+  }
+  // Build project
+  ...
+}
+----
+
+=== Signing Android build
+When an Android apk is created it can be signed using the provided `signAndroidApks` function.
+
+[source,groovy]
+----
+node("android"){
+  // Build the apk
+  ...
+  stage("Sign"){
+    if (params.BUILD_CONFIG == 'release') {
+        signAndroidApks (
+          keyStoreId: "${params.BUILD_CREDENTIAL_ID}",
+          keyAlias: "${params.BUILD_CREDENTIAL_ALIAS}",
+          apksToSign: "**/*-unsigned.apk",
+          // uncomment the following line to output the signed APK to a separate directory as described above
+          // signedApkMapping: [ $class: UnsignedApkBuilderDirMapping ],
+        )
+    } else {
+      println('Debug Build - Using default developer signing key')
+    }
+  }
+  ...
+}
+----
+
+The `signAndroidApks` function takes the following parameters.
+
+.signAndroidApks options
+|===
+| Parameter | Description
+
+| keyStoreId
+| Keystore ID. This will be provided through the `BUILD_CREDENTIAL_ID` parameter.
```

Comment:
Add a note here to explain that this parameter is only provided if the app is built from RHMAP